packages/dev/docs/pages/releases/2023-04-04.mdx

```diff
@@ -0,0 +1,96 @@
+* Fixing two broken links in docs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/4286)
+* Update Slider docs to match comment in table - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4301)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+* **TagGroup** (RC)
+  * Update useTagGroup and useTag returned prop names - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4156)
+  * TagGroup: Prevent focus of remove button via useTag - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4237)
+  * TagGroup: don't remove tags on Space press - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4240)
+  * Localize tag count - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4257)
+  * TagGroup: focus ring does not use Highlight color with forced-colors: active - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4198)
+  * Fix TagGroup maxRows layout issues - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4041)
+  * useTagGroup: add accessibility description and update 'remove' translation string - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4272)
+  * useTag: remove isFocused prop - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4291)
+  * TagGroup: fix types and item render - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4295)
+  * Add docs for useTagGroup - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4154)
+* **TableView column resizing** (RC)
+  * Prevent keyboard specific Table column resizing announcement on iOS Voiceover - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4232)
+  * Refine table column resize hook props and returned values  - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4254)
+  * Add custom cursor icons back in TableView column resizing - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3649) and [PR](https://github.com/adobe/react-spectrum/pull/4276)
+  * Automatically providing visually hidden styles to the table column resizer input - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4296)
+  * Table column resizing aria docs - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3982)
+* **ActionBar** (RC)
+* **Toast** (beta)
```

Comment:
Both of these had no code changes, but we decided to advance them according to the release plan (not sure about toast). Figured we should include them here anyway?